modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -309,7 +311,7 @@ public class IgniteImpl implements Ignite {
     private final ClusterInitializer clusterInitializer;
 
     /** Replica manager. */
-    private final ReplicaManager replicaMgr;
+    private ReplicaManager replicaMgr = null;
```

Comment:
Why? It's still doesn't change below

Reply:
Don't remember a reason for this. I return it back.